### PR TITLE
Factor out a few more things

### PR DESCRIPTION
**Pull request by chadwhitacre**
Part of #795.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -265,68 +265,14 @@ for topic in $NEEDED_KAFKA_TOPICS; do
...
-  $dc up -d --remove-orphans $($dc config --services | grep -v -E '^(nginx|relay)$')
-  $dc exec -T nginx service nginx reload
-
-  docker run --rm --network="${COMPOSE_PROJECT_NAME}_default" alpine ash \
-    -c 'while [[ "$(wget -T 1 -q -O- http://web:9000/_health/)" != "ok" ]]; do sleep 0.5; done'
-
-  # Make sure everything is up. This should only touch relay and nginx
-  $dc up -d
-  echo "${_endgroup}"
+  ./install/restart-carefully.sh
```
Comment (BYK) [inline install.sh:275]: lol at the file name

File: install.sh [hunk 2]
```diff
@@ -265,68 +265,14 @@ for topic in $NEEDED_KAFKA_TOPICS; do
...
-SENTRY_DATA_NEEDS_MIGRATION=$(docker run --rm -v sentry-data:/data alpine ash -c "[ ! -d '/data/files' ] && ls -A1x /data | wc -l || true")
-if [[ -n "$SENTRY_DATA_NEEDS_MIGRATION" ]]; then
-  # Use the web (Sentry) image so the file owners are kept as sentry:sentry
-  # The `\"` escape pattern is to make this compatible w/ Git Bash on Windows. See #329.
-  $dcr --entrypoint \"/bin/bash\" web -c \
-    "mkdir -p /tmp/files; mv /data/* /tmp/files/; mv /tmp/files /data/files; chown -R sentry:sentry /data"
-fi
-echo "${_endgroup}"
-
+./install/ensure-postgresql-version.sh
```
Comment (BYK) [inline install.sh:268]: Should probably be `upgrade postgres`

File: install/set-up-database.sh [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+source "$(dirname $0)/_lib.sh"
+
+echo "${_group}Setting up database ..."
```
Comment (BYK) [inline install/set-up-database.sh:4]: We should probably say `Setting up / migrating` here and in the file name.

**Review by BYK — APPROVED** [review]

File: install/set-up-database.sh (hunk 3 above)
Comment (chadwhitacre) [reply install/set-up-database.sh]: Done in 676e3b2ddc04bf6af65d3339f1924630e4e7dbdc.

File: install.sh (hunk 2 above)
Comment (chadwhitacre) [reply install.sh]: Done in 676e3b2ddc04bf6af65d3339f1924630e4e7dbdc.

**Pull request merged**